lib/javascripts/renderer.js

```diff
@@ -78,30 +25,24 @@ Renderer.prototype.nextLevelScreen = function(){
 Renderer.prototype.drawPlanes = function(planes) {
   let context = this.context;
   planes.forEach(function(plane){
-    plane.rectangle();
-    plane.planeWings();
-    context.fillStyle = "red";
-    var xMid = plane.x + (plane.width/2);
-    context.beginPath();
-    drawFuselage(context,plane);
-    drawNorthTriangle(context,xMid,plane);
-    drawSouthTriangle(context,xMid,plane);
-    if (plane.type === "player" && plane.class === "small") {
-      context.fillStyle = 'purple';
-      drawPlayerWings(context,xMid,plane);
-      drawPlayerTail(context,xMid,plane);
-    } else if (plane.type === "computer" && plane.class === "small") {
-      context.fillStyle = 'green';
-      drawComputerWings(context,xMid,plane);
-      drawComputerTail(context,xMid,plane);
-    } else if (plane.class === "boss") {
-      context.fillStyle = 'orange';
-      drawBossComputerWings(context,xMid,plane);
-      drawComputerTail(context,xMid,plane);
-      context.fillStyle = "red";
-      context.font = "bold 16px Arial";
-      context.fillText("Boss Lives: " + String(plane.hitCounterLimit - plane.hitCounter), 575, 485);
-    }
+      plane.rectangle();
+      plane.planeWings();
+      if (plane.type === "player" && plane.class === "small") {
+	  var img = new Image();
+	  img.src = "lib/player_plane.svg";
+	  context.drawImage(img, plane.x - 22, plane.y - 5, 500, 500);
+      } else if (plane.type === "computer" && plane.class === "small") {
+	  var img = new Image();
+	  img.src = "lib/computer_plane.svg";
+	  context.drawImage(img, plane.x - 22, plane.y - 35, 500, 500);
+      } else if (plane.class === "boss") {
+	  var img = new Image();
```

Comment:
@theonlyrao I notice that we're calling `var img = new image()` no matter what route it takes. Can we do that line before the if/else block? Otherwise it looks great.